pkg/api/v1beta3/dynakube/telemetryservice/props.go

```diff
@@ -1,37 +1,32 @@
 package telemetryservice
 
-type Protocol string
+import "github.com/Dynatrace/dynatrace-operator/pkg/otelcgen"
 
 const (
 	nameSuffix = "-telemetry"
-
-	OtlpProtocol   Protocol = "otlp"
-	ZipkinProtocol Protocol = "zipkin"
-	JaegerProtocol Protocol = "jaeger"
-	StatsdProtocol Protocol = "statsd"
 )
 
-func KnownProtocols() []Protocol {
-	return []Protocol{
-		OtlpProtocol,
-		ZipkinProtocol,
-		JaegerProtocol,
-		StatsdProtocol,
+func KnownProtocols() []otelcgen.Protocol {
```

Comment:
you can replace it with: `otelcgen.RegisteredProtocols`

Reply:
done